charged-ieee/lib.typ

```diff
@@ -45,21 +45,23 @@
   show figure.where(kind: table): set figure.caption(position: top)
   show figure.where(kind: table): set text(size: 8pt)
   show figure.where(kind: table): set figure(numbering: "I")
+  show figure.where(kind: table): set figure.caption(separator: [\ ])
```

Comment:
Let's group this up with the `show figure.where(kind: table): set figure.caption` a bit before.